crates/core/tedge_mapper/src/main.rs

```diff
@@ -111,11 +124,88 @@ async fn main() -> anyhow::Result<()> {
         )?);
     }
 
+    let mut mqtt_actor = get_mqtt_actor(component.session_name(), &config).await?;
+    let mut runtime = Runtime::try_new(runtime_events_logger).await?;
+    let mut signal_actor = SignalActor::builder();
     if mapper_opt.init {
         component.init(&mapper_opt.config_dir).await
     } else if mapper_opt.clear {
         component.clear_session().await
+    } else if component.session_name().eq("tedge-mapper-az") {
+        let add_time_stamp = config.query(AzureMapperTimestamp)?.is_set();
+        let clock = Box::new(WallClock);
+        let size_threshold = az_mapper_ext::size_threshold::SizeThreshold(255 * 1024);
+        let az_converter = AzureConverter::new(add_time_stamp, clock, size_threshold);
```

Comment:
I would introduce a `run_actor()` function to make the code clearer.

This would also help to:
- avoid launching the runtime when useless
- share the code for all the actors used by all the mappers.